testing.Dockerfile

```diff
@@ -1,12 +1,21 @@
+FROM golang:1.18 as builder
```

Comment:
These changes allows us to fix the golang tests that we have had on our backlog for a while